admin/lib/solidus_admin/system/providers/main_nav.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "solidus_admin/container"
+require "solidus_admin/main_nav_item"
+
+module SolidusAdmin
+  module Providers
+    Container.register_provider("main_nav") do
+      start do
+        container.namespace("main_nav") do
+          register("first_item", MainNavItem.new(title: "First item", position: "10"))
```

Comment:
Do you envision any problem with when we'll need to use I18n here for the title?

Reply:
I don't think so, as `I18n` acts globally. Alternatively, we could use the I18n key and translate from the component.